+NeuroAnalysis/+Base/ApplyFunctions.m

```diff
@@ -14,6 +14,9 @@
 end
 
 if isparallel
+    if isempty(gcp('nocreate'))
+        parpool; % start the parallel pool
+    end
```

Comment:
I remember Matlab will init parallel pool automatically if it hasn't been created. seems to me no need to do explicit works.